### ISSUE TITLE
Installation issues

### ISSUE THREAD

**Issue by garemoko** [LearningLocker/learninglocker#105]
Hi,
I managed to get Learning Locker installed on my laptop this lunchtime, but I've run into two issues:
1. I seem to have two copies of the code, so my server path is
   http://localhost/learninglocker/learninglocker/server.php
   The first learning locker folder has some, but not all of the code. 
2. I'm not getting any css, javascript or the logo. Is this intentional? It doesn't appear to be in the folder at all. 

Any ideas what I could have done wrong? 

Andrew

p.s. these instructions were very helpful: http://williamjxj.wordpress.com/2012/06/04/install-mongodb-with-windows-xampp/

**Comment by jamarr81** [comment]
I just ran into this same issue today. My environment is a little different in that I'm running nginx with php5-fpm on ubuntu. The problem for me was that if I accessed /index.php directly, I'm being redirected to /index.php/site at which point no Dashboard content (dashboard, settings, LRSs, Users, etc.) is visible. However, if I access / (root) by itself I'm redirected to /site at which point everything works.
